tests/test_install.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+set -e
+
+# Setup env
+INSTALL_PREFIX=$(cat /tmp/install_prefix)
```

Comment:
Can we pass this as the first CLI arg, rather than reading from file again? Makes this script more useful if we can use to test install directories in arbitrary places, without following the CI's restricted setup step.